### ISSUE TITLE
Extracting other resource types

### ISSUE THREAD

**Issue by clubside** [zeiban/defiance-tools#2]
Howdy! Your tool has been invaluable for my Defiance 2050 Guide project ([http://vgpavilion.com/defiance-2050/]). Unfortunately it opened up some possibilities I hadn't considered and now want to explore further lol

I can take the output of ski2obj into Adobe Dimension and render outfits I don't have and could never get. Wish I could figure out which textures are used to overlay the base hair eyes and skin but I'm sure I can find them eventually. What I'm really looking for are the skeleton and animation files. Since Defiance uses the Gamebryo engine I was hoping the mod scene around the Elder Scrolls games would help me figure things out but wading through the waddump has been difficult.

I plan on doing some more research on nif files ([http://www.niftools.org/]) which seem to be the heart of the engine's rendering pipeline but I'm not sure that's how these other 3D assets are stored.

Bottom line, it'd be cool to use the skeleton's for manual posing before static renders as well as live WebGL views showing both static and active animations.
